src/stores/WalletStore.ts

```diff
@@ -42,19 +41,26 @@ class WalletStore extends PersistableStore {
     return web3Modal.cachedProvider
   }
 
-  async connect(clearCachedProvider = false) {
+  async connect(clearCachedProvider = false, checkNetwork = false) {
```

Comment:
Let's create an object here, because calling `walletstore.connect(true, true)` is strange